### DISCUSSION TITLE
Error running config

### DISCUSSION THREAD

**Discussion by I-Vuk** [hrsh7th/nvim-cmp#1339] [Q&A]
Hi, 
I'm trying to install nvim-CMP for neovim using packer, but I am getting an error.
The ERROR: packer.nvim: Error running config for nvim-CMP: [string "..."]: 0: attempt to index a boolean value

Does anyone have a solution for this?

NeoVim version: 
nvim v 0.8.1.

OS version:
NAME="Pop!_OS"
VERSION="22.04 LTS"

**Comment by Shougo** [comment]
Hm...  You should upload the config.  I think your configuration is wrong.

**Reply by I-Vuk** [reply]
[cmp-config.zip](https://github.com/hrsh7th/nvim-cmp/files/10164787/cmp-config.zip)

Here are the files. 3 files in total: Plugins.lua which uses Packer to get the lua-cmp (among others), cmp.lua which contains the girth of the config for the lua-cmp, and init.lua which is the main file and imports the other 2.

**Reply by Shougo** [reply]
Hm...  Can you create the minimal config without `packer.nvim`?

For example:

```vim
set rtp+=~/src/nvim-cmp
set rtp+=~/src/cmp-buffer
set rtp+=~/src/cmp-nvim-lsp
set rtp+=~/src/nvim-lspconfig

set completeopt=menuone,noselect

lua <<EOF
  -- Setup nvim-cmp.
  local cmp = require'cmp'

  cmp.setup({
    mapping = {
      ['<CR>'] = cmp.mapping.confirm({ select = true }),
      ['<tab>'] = cmp.mapping(cmp.mapping.confirm({ select = true }), { 'i', 's', 'c' }),
    },
    sources = {
      { name = 'nvim_lsp' },
      { name = 'buffer' },
    },
  })
EOF

lua << EOF
local capabilities = require('cmp_nvim_lsp').update_capabilities(vim.lsp.protocol.make_client_capabilities())

require'lspconfig'.gopls.setup{}
EOF

runtime! after/plugin/*.lua
```
